pipelines/package_names.go

```diff
@@ -0,0 +1,68 @@
+package pipelines
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/grafana/grafana-build/executil"
+)
+
+type TarFileOpts struct {
+	Version      string
+	BuildID      string
+	IsEnterprise bool
+	Distro       executil.Distribution
+}
+
+// TarFilename returns a file name that matches this format: {grafana|grafana-enterprise}_{version}_{os}_{arch}_{build_number}.tar.gz
+func TarFilename(opts TarFileOpts) string {
```

Comment:
Nice move here, I actually have been thinking of how to reduce arguments to functions that take too many args and using an Opts struct type seems like a great idea.

Reply:
One drawback is that if we add fields to the the opts struct the compiler won't tell us in places where we forgot to populate them.